.github/workflows/docs.yml

```
@@ -158,6 +176,23 @@ jobs:
           publish_dir: doc/_build/html/
           cname: docs.pyvista.org
 
+      - name: Remove non-multi version folder
+        run: rm -rf doc
+
+      - name: Deploy on main multiversion
+        if: github.event_name == 'push'
+        uses: pyansys/actions/doc-deploy-dev@main
+        with:
+            token: ${{ secrets.GITHUB_TOKEN }}
+            decompress_artifact: true
+     
+      - name: Deploy on release multiversion
+        if: github.event_name == 'push' && contains(github.ref, 'refs/tags')
+        uses: pyansys/actions/doc-deploy-stable@main
+        with:
```

Same comment about the `CNAME` applies here:

```suggestion
        with:
          cname: ${{ env.DOCUMENTATION_CNAME }}
```